src/api-types/projects/projects.ts

```diff
@@ -0,0 +1,24 @@
+import { Projects } from '@/types/projects';
+
+export type ProjectsApiModel = {
```

Comment:
```suggestion
export type ProjectApiModel = {
```
And rename Projects to Project in all places where it's needed